Automattic/wp-calypso
# Add Sensei checklist task

#### Proposed Changes

* Adds a Sensei task to the onboarding checklist

#### Testing Instructions

* Check out this patch to your WPCOM sandbox: D98792-code
* Make sure to point `public-api.wordpress.com` to your Sandbox IP through your hosts file
* Go through the Sensei Flow and ensure you see the checklist item on the onboarding checklist
* Make sure the CTA link sends you to the Sensei Home page.
* Note:  currently it does not complete.

<img width="1000" alt="Screenshot 2023-01-19 at 3 14 03 PM" src="https://user-images.githubusercontent.com/3220162/213583070-1f801987-130f-43f2-b144-9e7026678da4.png">
